azure-csharp/${PROJECT}.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Pulumi" Version="3.*" />
-    <PackageReference Include="Pulumi.AzureNative" Version="1.*" />
+    <PackageReference Include="Pulumi.AzureNative" Version="2.0.0" />
```

Comment:
Why is this not `"2.*"`, like `"1.*"` before?

Reply:
good catch!